aman/src/components/Button.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+const Button = ({ buttonName }) => {
+  return (
+    <div>
+      <button
+        className="bg-blue-900 w-96 h-11 mt-10 text-white text-sm font-medium rounded-lg max-w-sm"
```

Comment:
Here since you have defined a width of 96 which means 384px and then also defined max-w-sm which means 384px this is redundancy. Check this.

Reply:
Yes it is redundancy , i'have removed that .